ctfcli/cli/challenges.py

```diff
@@ -51,27 +51,35 @@ def templates(self):
 
         Templates().list()
 
-    def add(self, repo):
+    def add(self, repo, category=None):
```

Comment:
This is better phrased as `folder`. It should be something more like `ctf challenge add <challenge> <folder>` which would behave a lot like `git clone <url> <folder>`

Reply:
But I would probably try to get this change to work without the parameter first before adding it on.